deploy/main.py

```diff
@@ -31,9 +31,10 @@
 # General configuration
 #
 
-conf["viur.validApplicationIDs"] = [
-	"{{app_id}}"
-]
+conf["viur.validApplicationIDs"] = \
+    [
+        "{{app_id}}"
+    ]
```

Comment:
I don't see a problem with the old code. The `\` often does not look really nice.